neuralpredictors/data/datasets/base.py

```diff
@@ -309,6 +310,9 @@ def __init__(self, dirname, *data_keys, transforms=None, use_cache=True, output_
         self.rename_output = bool(output_rename)
         self._output_rename = output_rename
         renamed_keys = [output_rename.get(k, k) for k in data_keys]
+        if output_rename:
+            # Output Point needs to be available for pickle in the global context 
+            global OutputPoint
```

Comment:
Does this work? I thought that the named tuple itself should be available in the global context. Have you tested it?
Also, If `output_rename` is `None` then `self.data_point` is used (next line) which is also a named tuple and should again be available in the global context. Or am I missing something?